src/tools/Makefile.am

```diff
@@ -22,26 +22,42 @@ do_subst = $(SED) \
 	   -e 's,[@]X509DIR[@],$(X509DIR),g'
 
 MAINTAINERCLEANFILES = $(srcdir)/Makefile.in $(srcdir)/versioninfo-tools.rc $(srcdir)/versioninfo-opensc-notify.rc
-EXTRA_DIST = Makefile.mak versioninfo-tools.rc.in versioninfo-opensc-notify.rc.in goid-tool.ggo.in npa-tool.ggo.in opensc-notify.ggo.in egk-tool.ggo.in opensc-asn1.ggo.in org.opensc.notify.desktop.in exe.manifest pkcs11-register.desktop.in org.opensc-project.mac.pkcs11-register.plist.in org.opensc-project.mac.opensc-notify.plist.in
+EXTRA_DIST = Makefile.mak versioninfo-tools.rc.in versioninfo-opensc-notify.rc.in goid-tool.ggo.in npa-tool.ggo.in opensc-notify.ggo.in egk-tool.ggo.in pkcs11-register.ggo.in opensc-asn1.ggo.in org.opensc.notify.desktop.in exe.manifest pkcs11-register.desktop.in org.opensc-project.mac.pkcs11-register.plist.in org.opensc-project.mac.opensc-notify.plist.in
 
 noinst_HEADERS = util.h fread_to_eof.h \
-	egk-tool-cmdline.h goid-tool-cmdline.h npa-tool-cmdline.h \
-	opensc-asn1-cmdline.h opensc-notify-cmdline.h pkcs11-register-cmdline.h \
 	openpgp-tool-helpers.h
-bin_PROGRAMS = opensc-tool opensc-explorer opensc-asn1 \
-	pkcs15-tool pkcs15-crypt pkcs11-tool pkcs11-register \
-	cardos-tool eidenv openpgp-tool iasecc-tool egk-tool goid-tool \
+bin_PROGRAMS = opensc-tool opensc-explorer \
+	pkcs15-tool pkcs15-crypt pkcs11-tool \
+	cardos-tool eidenv openpgp-tool iasecc-tool \
 	dtrust-tool
 if ENABLE_OPENSSL
 bin_PROGRAMS += cryptoflex-tool pkcs15-init netkey-tool piv-tool \
 	westcos-tool sc-hsm-tool dnie-tool gids-tool
+endif
+
+if ENABLE_GENGETOPT
+bin_PROGRAMS += egk-tool goid-tool opensc-asn1 pkcs11-register
+
+if ENABLE_NOTIFY
+bin_PROGRAMS += opensc-notify
+endif
```

Comment:
Can you move these two blocks below the `if ENABLE_OPENSSL` to avoid closing and opening the `ENABLE_OPENSSL` block before and after this?